### ISSUE TITLE
"Why can't I display GFlops when initializing the YOLO class model?"

### ISSUE THREAD

**Issue by pengsmallcute** [ultralytics/ultralytics#6974]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

you change the model scale is n

                   from  n    params  module                                       arguments                     
  0                  -1  1       448  ultralytics.nn.modules.conv.GhostConv        [3, 16, 3, 2]                 
  1                  -1  1      2768  ultralytics.nn.modules.conv.GhostConv        [16, 32, 3, 2]                
  2                  -1  1      7360  ultralytics.nn.modules.block.C2f             [32, 32, 1, True]             
  3                  -1  1     10144  ultralytics.nn.modules.conv.GhostConv        [32, 64, 3, 2]                
  4                  -1  2     49664  ultralytics.nn.modules.block.C2f             [64, 64, 2, True]             
  5                  -1  1     38720  ultralytics.nn.modules.conv.GhostConv        [64, 128, 3, 2]               
  6                  -1  2    197632  ultralytics.nn.modules.block.C2f             [128, 128, 2, True]           
  7                  -1  1    151168  ultralytics.nn.modules.conv.GhostConv        [128, 256, 3, 2]              
  8                  -1  1    460288  ultralytics.nn.modules.block.C2f             [256, 256, 1, True]           
  9                  -1  1    164608  ultralytics.nn.modules.block.SPPF            [256, 256, 5]                 
 10                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']          
 11             [-1, 6]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 12                  -1  1    148224  ultralytics.nn.modules.block.C2f             [384, 128, 1]                 
 13                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']          
 14             [-1, 4]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 15                  -1  1     37248  ultralytics.nn.modules.block.C2f             [192, 64, 1]                  
 16                  -1  1     36992  ultralytics.nn.modules.conv.Conv             [64, 64, 3, 2]                
 17            [-1, 12]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 18                  -1  1    123648  ultralytics.nn.modules.block.C2f             [192, 128, 1]                 
 19                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]              
 20             [-1, 9]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 21                  -1  1    493056  ultralytics.nn.modules.block.C2f             [384, 256, 1]                 
 22        [15, 18, 21]  1    754237  ultralytics.nn.modules.head.Detect           [15, [64, 128, 256]]          
ghost_conv_v8n summary: 245 layers, 2823917 parameters, 2823901 gradients

### Additional

my code is this:
from ultralytics import YOLO

# Load a model
model = YOLO(model= r'D:\code _vs\yolov8\ultralytics-main\ultralytics\cfg\models\v8\ghost_conv_v8n.yaml')  # build a new model from YAML

**Comment by glenn-jocher** [comment]
@pengsmallcute hello! The display of GFLOPs (Giga Floating Point Operations per Second) during the initialization of the YOLO class model is not shown by default in the output. The table you're seeing is related to layer-by-layer details like module types, parameters, and module arguments, but GFLOPs are not included here.

If keeping track of the GFLOPs is crucial for your work, you may want to use profiling tools or modify the source code to include GFLOP calculations during the model summary output. However, do remember that modifying the source will mean diverging from the standard repository updates which could complicate future merging.

For further insights into YOLOv8's capabilities and usage, including more advanced configurations, you might want to check out the our comprehensive documentation over at [Ultralytics Docs](https://docs.ultralytics.com). Our Docs provide extensive guides and references that could help you optimize and understand all facets of YOLOv8's performance 🚀.
